src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -195,30 +194,35 @@ void CascadiaSettings::_CreateDefaultProfiles()
     cmdProfile.SetAcrylicOpacity(0.75);
     cmdProfile.SetUseAcrylic(true);
 
-    auto powershellProfile{ _CreateDefaultProfile(L"PowerShell") };
+    auto powershellProfile{ _CreateDefaultProfile(L"Windows PowerShell") };
+    powershellProfile.SetCommandline(L"powershell.exe");
+    powershellProfile.SetStartingDirectory(DEFAULT_STARTING_DIRECTORY);
+    powershellProfile.SetColorScheme({ L"Campbell" });
+    powershellProfile.SetDefaultBackground(POWERSHELL_BLUE);
+    powershellProfile.SetUseAcrylic(false);
+
     // If the user has installed PowerShell Core, we add PowerShell Core as a default.
     // PowerShell Core default folder is "%PROGRAMFILES%\PowerShell\[Version]\".
-    std::wstring psCmdline = L"powershell.exe";
     std::filesystem::path psCoreCmdline{};
-    if (_IsPowerShellCoreInstalled(L"%ProgramFiles%", psCoreCmdline))
+    if (_isPowerShellCoreInstalled(psCoreCmdline))
     {
-        psCmdline = psCoreCmdline;
+        Profile pwshProfile{ _CreateDefaultProfile(L"PowerShell Core") };

```

Comment:
```suggestion
        auto pwshProfile{ _CreateDefaultProfile(L"PowerShell Core") };
```

You used auto on the one above...